mcs/class/System.Drawing/System.Drawing/gdipFunctions.cs

```diff
@@ -979,22 +988,22 @@ static internal void CheckStatus (Status status)
 		internal static extern Status GdipBitmapSetPixel (IntPtr bmp, int x, int y, int argb);
 
 		// Image functions
-		[DllImport(GdiPlus, CharSet=CharSet.Auto)]
+		[DllImport(GdiPlus, CharSet=GdiPlusCharSet)]
 		internal static extern Status GdipLoadImageFromFile ( [MarshalAs(UnmanagedType.LPWStr)] string filename, out IntPtr image );
```

Comment:
I'd prefer making the CharSet Unicode explicitly where possible.

The cases where this isn't possible are where LOGFONT is involved, correct?